lib/mempool/mempool.js

```diff
@@ -193,13 +210,6 @@ class Mempool extends EventEmitter {
    */
 
   async _addBlock(block, txs, view) {
-    if (this.map.size === 0
```

Comment:
Why remove this ?

Reply:
See https://github.com/handshake-org/hsd/pull/319 

I don't think there was any reason for this to be here besides early-bcoin-style laziness and maybe even an accidental copy-paste from `_removeBlock()`where it used to make more sense, to just avoid complicated mempool issues.

What the function does is resolve orphans whose missing parents were just confirmed. There is no reason to NOT do that just because the current mempool is empty.